includes/media.php

```diff
@@ -140,12 +142,14 @@ function pods_image_url( $image, $size = 'thumbnail', $default = 0, $force = fal
 	$default = pods_image_id_from_field( $default );
 
 	if ( 0 < $id ) {
-		if ( $force ) {
+		if ( $force && 'full' !== $size ) {
 			$full = wp_get_attachment_image_src( $id, 'full' );
-			$src  = wp_get_attachment_image_src( $id, $size );
+			if ( ! empty( $full[0] ) ) {
+				$src  = wp_get_attachment_image_src( $id, $size );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 96b538570746d1f44470979393e3d3f5</sub>